src/components/paperManager.js

```diff
@@ -93,4 +93,8 @@ export default class PaperManager {
   setStateValid() {
     this.paper.drawBackground({ color: defaultNodeColor });
   }
+
+  hasScheduledUpdates() {
+    return this.#paper._updates.priorities.some(updates => !util.isEmpty(updates));
```

Comment:
@chipit24 found this solution online. Some browsers (Safari...) were not properly detecting if updates needed to be rendered from the 'isRendering' property. This code works properly and gets around that issue. We should maybe look at using this more consistently in the project.

Reply:
I didn't really find it, I asked for it since I created that issue in the jointjs repo :p I find they are quick to reply if you post an issue at https://github.com/clientIO/joint/issues.

Reply:
In either case, you take the credit Rob. :)